Lib9c/Action/AccountStateDeltaExtensions.cs

```diff
@@ -872,7 +872,10 @@ public static ShopState GetShopState(this IAccountStateDelta states)
             return (arenaInfoAddress, arenaInfo, isNew);
         }
 
-        public static bool TryGetStakeState(this IAccountStateDelta states, Address agentAddress, out StakeState stakeState)
+        public static bool TryGetStakeState(
+            this IAccountStateDelta states,
+            Address agentAddress,
+            out StakeState stakeState)
```

Comment:
It's minor but I hope the part formatting code separated as other pull request or commit.